### PR TITLE
Documentation for the PyPI Integration

### PR DESCRIPTION
**Pull request by MinuraPunchihewa**
## Description

This PR adds the documentation for the PyPI integration.

Fixes https://github.com/mindsdb/mindsdb/issues/8043

## Type of change

- [X] 📄 This is a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Ensure a new section for the PyPI integration is available in the docs site

## Checklist:

- [X] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [X] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: docs/integrations/app-integrations/pypi.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+## Connection
+
+This handler is implemented using the standard Python `requests` library. It is used to connect to the RESTful service that [pypistats.org](https://pypistats.org) is serving.
+
+There are no connection arguments required to initialize the handler.
+
+To connect to PyPI using MindsDB, the following CREATE DATABASE statement can be used:
+
+```sql
+CREATE DATABASE mediawiki_datasource
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/pypi.mdx:21]: Let's call it `pypi_datasource`.

**Comment by martyna-mindsdb** [comment]
@MinuraPunchihewa 
Please update the data source name. The rest looks good.

File: docs/integrations/app-integrations/pypi.mdx (hunk 1 above)
Comment (MinuraPunchihewa) [reply docs/integrations/app-integrations/pypi.mdx]: Oh, my bad, @martyna-mindsdb. It's done.

**Comment by MinuraPunchihewa** [comment]
It's done, @martyna-mindsdb. I've also added another small tip.

**Pull request merged**